streamz/dataframe/tests/test_cudf_dataframes.py

```diff
@@ -33,23 +32,23 @@ def client():
         client.close()
 
 
-@pytest.fixture(params=['core', 'dask'])
+@pytest.fixture(params=["core", "dask"])
```

Comment:
Are we standardizing on double quotes? From what I've seen, Python code has always used single quotes for strings unless there was a reason to do otherwise.

Reply:
Sure, will fix this.

Reply:
Fixed

Reply:
If we are going to go this way we should commit to black and be done with it IMO.